Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable-slim
+MAINTAINER Vincent Giersch <vincent@flat.io>
+
+RUN mkdir /faust
+WORKDIR /faust
+COPY . /faust
```

Comment:
Yes, that's a way to reduce the size of the image by 2 since we can install & remove the build dependencies in a single Docker Layer. We can do like the original version but the result of the image is much bigger (but we can take the advantage of the cache when developing).